src/Microsoft.NetCore.Analyzers/UnitTests/Tasks/UseValueTaskCorrectlyTests.cs

```diff
@@ -0,0 +1,1274 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Testing;
+using Microsoft.CodeAnalysis.VisualBasic;
+using Test.Utilities;
+using Xunit;
+
+namespace Microsoft.NetCore.Analyzers.Tasks.UnitTests
+{
+    public class UseValueTasksCorrectlyTests : DiagnosticAnalyzerTestBase
```

Comment:
@sharwell and @Evangelink are working towards removal of `DiagnosticAnalyzerTestBase` in favor of new Microsoft.CodeAnalysis.Testing library, see https://github.com/dotnet/roslyn-analyzers/pull/3052. We may want to ensure new tests are directly written with the new library, so there are no merge conflicts with that PR.